ds_judgements_public_ui/sass/includes/_footer.scss

```diff
@@ -62,23 +62,60 @@
     }
   }
 
+  &__ojl-paragraph {
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__ojl-logo {
+    width: 50px;
+    height: 31px;
+    margin-right: $space-3;
+    margin-bottom: -$space-4;
+
+    @media (width <= 479px) {
+      margin-bottom: 0;
+    }
+  }
+
   &__base-paragraph {
-    margin-top: $space-6;
+    margin-top: $space-8;
     font-size: $typography-sm-text-size;
     line-height: 1.2rem;
     text-align: left;
 
+    @media (min-width: $grid-breakpoint-medium) {
+      font-size: $typography-sm-text-size;
+    }
+
+    @media (min-width: $grid-breakpoint-extra-large) {
+      text-align: left;
+    }
+  }
+
+  &__base-paragraph-second {
```

Comment:
This feels like it's a touch over-specific, and could be achieved using `:first` or similar